docs/compatibility.md

```diff
@@ -115,9 +115,11 @@ Distrobox guests tested successfully with the following container images:
 |    Distro  |    Version | Images    |
 | --- | --- | --- |
 | AlmaLinux (Toolbox) | 8 <br> 9 | quay.io/toolbx-images/almalinux-toolbox:8 <br> quay.io/toolbx-images/almalinux-toolbox:9 <br> quay.io/toolbx-images/almalinux-toolbox:latest |
-| Alpine (Toolbox) | 3.16 <br> 3.17 <br> 3.18 <br> 3.19 <br> 3.20 <br> edge | quay.io/toolbx-images/alpine-toolbox:3.16 <br> quay.io/toolbx-images/alpine-toolbox:3.17 <br> quay.io/toolbx-images/alpine-toolbox:3.18 <br> quay.io/toolbx-images/alpine-toolbox:3.19 <br> quay.io/toolbx-images/alpine-toolbox:3.20 <br> quay.io/toolbx-images/alpine-toolbox:edge <br> quay.io/toolbx-images/alpine-toolbox:latest |
+| Alpine (Toolbox) | 3.18 <br> 3.19 <br> 3.20 <br> 3.21 <br> edge <br> latest | quay.io/toolbx-images/alpine-toolbox:3.18 <br> quay.io/toolbx-images/alpine-toolbox:3.19 <br> quay.io/toolbx-images/alpine-toolbox:3.20 <br> quay.io/toolbx-images/alpine-toolbox:3.21 <br> quay.io/toolbx-images/alpine-toolbox:edge <br> quay.io/toolbx-images/alpine-toolbox:latest |
+| PostmarketOS (Toolbox) | 23.06 <br> 23.12 <br> 24.06 <br> 24.12 <br> edge <br> latest | quay.io/gmanka/pmos-toolbox:23.06 <br> quay.io/gmanka/pmos-toolbox:23.12 <br> quay.io/gmanka/pmos-toolbox:24.06 <br> quay.io/gmanka/pmos-toolbox:24.12 <br> quay.io/gmanka/pmos-toolbox:edge <br> quay.io/gmanka/pmos-toolbox:latest <br>
```

Comment:
Despite I'd love to label PostmarketOS as officially supported, these are just images released under your namespace in Quay. Since these images are end-to-end tested, I would appreciate something officially labeled.